packages/dds/sequence/src/defaultMap.ts

```diff
@@ -406,12 +411,11 @@ export class DefaultMap<T> {
 				assert(localValue !== undefined, 0x3f8 /* Local value expected on resubmission */);
 
 				const handler = localValue.getOpHandler(op.value.opName);
-				const { rebasedOp, rebasedLocalOpMetadata } = handler.rebase(
-					localValue.value,
-					op.value,
-					localOpMetadata,
-				);
-				this.submitMessage({ ...op, value: rebasedOp }, rebasedLocalOpMetadata);
+				const rebased = handler.rebase(localValue.value, op.value, localOpMetadata);
+				if (rebased !== undefined) {
```

Comment:
@connorskees  and @jzaffiro i took this a different way. i now allow the rebase function to return undefined, which means do not send an op, which is allowed on resubmit paths. i think this is better, as i don't think the client will process and op with undefined as the interval value correctly, so better to just not send it.